keyboards/zen/rev2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,129 @@
+#include QMK_KEYBOARD_H
+#include "quantum.h"
```

Comment:
This shouldn't be needed. it should be included by the keyboard's header file, which is defined as `QMK_KEYBOARD_H` at compile time. 

```suggestion
```

Reply:
Removed